src/main/webapp/app/assessment/athena.service.ts

```diff
@@ -0,0 +1,31 @@
+import { Injectable } from '@angular/core';
+import { HttpClient, HttpResponse } from '@angular/common/http';
+import { Observable, of, switchMap } from 'rxjs';
+import { TextBlockRef } from 'app/entities/text-block-ref.model';
+import { ProfileService } from 'app/shared/layouts/profiles/profile.service';
+
+type FeedbackSuggestionsResponseType = HttpResponse<TextBlockRef[]>;
+
+@Injectable({ providedIn: 'root' })
+export class AthenaService {
+    public resourceUrl = 'api/athena';
+
+    constructor(protected http: HttpClient, private profileService: ProfileService) {}
```

Comment:
codacy

Reply:
ESLint doesn't let me do this: The pre-commit hook always changes it back :/